radixdlt/src/test/java/com/radixdlt/network/addressbook/PeerManagerTest.java

```diff
@@ -15,13 +15,18 @@
  * language governing permissions and limitations under the License.
  */
 
-package org.radix.network2.addressbook;
```

Comment:
Pretty much everything in this file is one of:
1. Package moves
2. Constant renames
2. Remove Interface class
3. Checkstyle lines too long